app/assets/javascripts/validation.coffee

```diff
@@ -779,6 +780,52 @@ $(document).ready ->
 
             errors.push(error)
 
+  # These errors are not captured in the schema, they are business logic being
+  # enforced in the CMR. Since we cannot publish records with these conditions,
+  # we should provide that feedback to the user. Specifically, in the
+  # Additional Attribute the Value supplied by the user needs to conform to the
+  # Data Type the user selects. The checks in the errorPresent block are meant
+  # to APPROXIMATE those of CMR ingestion with the intention that there are
+  # likely some input values that the below logic will permit, and CMR will not,
+  # but never vice versa.
+  validateAdditionalAttributeValueField = (errors) ->
+
+    if $('#draft_additional_attributes').length > 0
+      floatRegex = new RegExp floatRegexString
+      intRegex = new RegExp intRegexString
+      boolRegex = new RegExp boolRegexString
+      dateRegex = new RegExp dateRegexString
+      timeRegex = new RegExp timeRegexString
+      dateTimeRegex = new RegExp dateTimeRegexString
+
+    $('#draft_additional_attributes').children('.multiple-item').each (index, element) ->
```

Comment:
another nitpick: I think this probably works fine, if it were me, I'd indent this to be nested under the check you just added.